### PR TITLE
Add add and remove buuton functions with dispact hook

### PR DESCRIPTION
**Pull request by Divinity-dev**
In this milestone of the project, i have done the following;
1. I have displayed books, received from the Bookslice.
1. I have added the a button with a function to call the addBook reducer witrh the useDispact hook.
2. I have added the a button with a function to call the removeBook reducer witrh the useDispact hook.
3. I have added the a button with a function to call the checkstatus reducer witrh the useDispact hook.
4. i disabled the camel-case ruse so as to let me use the item_id as stated in the instruction.

### CODE REVIEW

File: src/Components/Book.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+    <div id={item_id} className="Books">
       <div className="B1">
         <h4>Action</h4>
         <h2>{title}</h2>
         <h3>{author}</h3>
         <div className="comps">
           <h3>comments</h3>
-          <h3>remove</h3>
+          <button className="btn" type="button" onClick={() => dispatch(removeBook({ item_id }))}>remove</button>
+          <button className="btn" type="button" onClick={() => dispatch(checkStatus())}>check status</button>
```
Comment (DammyShittu) [inline src/Components/Book.js]: - [ ] I think this checkStatus dispatch here is meant for the categories component. When I visit the categories component and click the check status button, you should dispatch an action to the store and fetch the "under construction" string from the store and display it on the page. Kindly do that.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Divinity-dev,

## Highlights
✔️ Books addition functionality

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Noted and thank you. I was confused about this.

Sent from Yahoo Mail on Android 
 
  On Wed, Mar 8, 2023 at 4:43 PM, Adedamola ***@***.***> wrote:   


@DammyShittu requested changes on this pull request.

Hi @Divinity-dev,

Highlights

 ✔️ Books addition functionality

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️
 
Check the comments under the review.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In src/Components/Book.js:
>        <div className="B1">
         <h4>Action</h4>
         <h2>{title}</h2>
         <h3>{author}</h3>
         <div className="comps">
           <h3>comments</h3>
-          <h3>remove</h3>
+          <button className="btn" type="button" onClick={() => dispatch(removeBook({ item_id }))}>remove</button>
+          <button className="btn" type="button" onClick={() => dispatch(checkStatus())}>check status</button>
   
   -    
 I think this checkStatus dispatch here is meant for the categories component. When I visit the categories component and click the check status button, you should dispatch an action to the store and fetch the "under construction" string from the store and display it on the page. Kindly do that.

**Review by Davidosky007 — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 
* Good looking PR description 
* Proper use of Gitflow 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you for the feedback
    On Wednesday, March 8, 2023 at 08:42:40 PM GMT+1, David Bassey ***@***.***> wrote:  
 
 


@Davidosky007 approved this pull request.

Hi,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 
Highlights
   
   - No linter errors
   - Good looking PR description
   - Proper use of Gitflow

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**